### DISCUSSION TITLE
Can I use this action to deploy a standard jekyll page from /docs?

### DISCUSSION THREAD

**Discussion by marvk** [JamesIves/github-pages-deploy-action#573] [Q&A]
# Situation

I am using GitHub Pages to display a simple landing page for my project with direct download links to artifacts generated by a GitHub Action. This information is pulled from the GitHub API via this syntax in the standard GitHub Pages `index.md`:

    [Download Installer (Windows)]({{ site.github.releases[0].assets[1].browser_download_url }})

When I push to the master branch, I run an action that automatically releases new binaries as GitHub Releases. Unfortunately, the GitHub Pages page is being deployed at the same time, with the GitHub API still providing the previous release as the latest release.

### GitHub Pages configuration

[![enter image description here][1]][1]

### Deployment page:

[![enter image description here][2]][2]

# Question(s)

My idea was to use a GitHub Action *after* the release is complete to deploy a fresh page. Can I use this action to trigger what GitHub seems to do automagically after each push? That is, just deploying the standard jekyll page from `/docs`. And is there a way to disable the automatic deployment so that the page *only* gets deployed after a (successful) release.

  [1]: https://i.stack.imgur.com/2dK5e.png
  [2]: https://i.stack.imgur.com/Ezrqa.png

**Comment by JamesIves** [comment]
You can use GitHub Actions to push a workflow [when a release is created](https://github.com/JamesIves/github-pages-deploy-action/blob/dev/.github/workflows/publish.yml#L2-L4). It's probably a little bit overkill, but in theory you could push a random file update into the `docs` directory using this action on the creation of a release which in turn should trigger GitHub Pages to re-build the Jekyll page.

I haven't used Jekyll with GitHub Pages in a very long time so I'm not sure if there's a way to hook into the deployment job that GitHub generates automatically using actions.

**Reply by marvk** [reply]
Hello James, thank you for your answer. 

In the end, I was able to solve my problem with an API call.

See [this](https://stackoverflow.com/questions/24098792/how-to-force-github-pages-build) Stackoverflow thread.

**Reply by JamesIves** [reply]
Awesome, that's probably a better solution anyway.

Glad it worked out for you!
